mobsf/StaticAnalyzer/views/android/playstore.py

```diff
@@ -35,7 +43,21 @@ def get_app_details(app_dic, man_data):
         msg = f'Fetching Details from Play Store: {package_id}'
         logger.info(msg)
         append_scan_status(checksum, msg)
-        det = app(package_id)
+        proxies, verify = upstream_proxy('https', for_urllib=True)
+        proxy_handler = ProxyHandler(proxies)
+        if verify:
+            ssl_context = ssl.create_default_context()
+        else:
+            ssl_context = ssl._create_unverified_context()
+        https_handler = HTTPSHandler(context=ssl_context)
+        opener = build_opener(proxy_handler, https_handler)
+        install_opener(opener)
+        try:
+            with opener.open(Request(PLAYSTORE), timeout=5) as response:
+                if response.status == 200:
+                    det = app(package_id)
```

Comment:
nit: Add a comment that play store details is fetched only if playstore is reachable.